src/styles/Book.module.css

```diff
@@ -1,12 +1,16 @@
+:root {
+  --flex-display: flex;
+}
```

Comment:
- [ ] You did a good job working on the previous reviewer's requirements, however, this doesn't get rid of repetition, instead, you're repeating this same `:root` rule here and in your `App.css`. As suggested by the previous reviewer It would be great if you can find a way of reducing this duplication of code ♻️. Remember that not repeating pieces of code will keep your codebase clean and readable. 🤓 🛠️

From [HTML & CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md):
![image](https://user-images.githubusercontent.com/77069011/200000947-9e58c4d8-fed4-45f8-a822-7828c6e713f7.png)

In this case, you could do this with flex:

```CSS
.flex {
  display: flex;
}
```

And then, use it in your HTML elements where you need a flex container:

```HTML
<div class="flex"> This is a flex container </div>
```

Please, consider fixing this! 🤓